### PR TITLE
[CLI] add request permission e2e

### PR DESCRIPTION
**Pull request by 0xjinn**
### Test Plan

```
2023-08-11 09:28:41,201 - INFO - These tests passed:
2023-08-11 09:28:41,201 - INFO - test_metrics_accessible
2023-08-11 09:28:41,201 - INFO - test_init
2023-08-11 09:28:41,201 - INFO - test_config_show_profiles
2023-08-11 09:28:41,201 - INFO - test_account_fund_with_faucet
2023-08-11 09:28:41,201 - INFO - test_account_create
2023-08-11 09:28:41,201 - INFO - test_account_lookup_address
2023-08-11 09:28:41,201 - INFO - test_aptos_header_included
2023-08-11 09:28:41,201 - INFO - test_move_compile
2023-08-11 09:28:41,201 - INFO - test_move_compile_script
2023-08-11 09:28:41,202 - INFO - test_move_publish
2023-08-11 09:28:41,202 - INFO - test_move_run
2023-08-11 09:28:41,202 - INFO - test_move_view
2023-08-11 09:28:41,202 - INFO - test_stake_initialize_stake_owner
2023-08-11 09:28:41,202 - INFO - test_stake_add_stake
2023-08-11 09:28:41,202 - INFO - test_stake_withdraw_stake_before_unlock
2023-08-11 09:28:41,202 - INFO - test_stake_unlock_stake
2023-08-11 09:28:41,202 - INFO - test_stake_withdraw_stake_after_unlock
2023-08-11 09:28:41,202 - INFO - test_stake_increase_lockup
2023-08-11 09:28:41,202 - INFO - test_stake_set_operator
2023-08-11 09:28:41,202 - INFO - test_stake_set_voter
2023-08-11 09:28:41,202 - INFO - test_stake_create_staking_contract
2023-08-11 09:28:41,202 - INFO - test_stake_request_commisson
2023-08-11 09:28:41,202 - INFO - test_node_show_validator_set
2023-08-11 09:28:41,202 - INFO - test_node_update_consensus_key
2023-08-11 09:28:41,202 - INFO - test_node_update_validator_network_address
2023-08-11 09:28:41,202 - INFO - test_account_rotate_key
2023-08-11 09:28:41,202 - INFO - All tests passed!
```

### CODE REVIEW

File: crates/aptos/e2e/cases/stake.py [hunk 1]
```diff
@@ -356,3 +356,63 @@ def test_stake_withdraw_stake_after_unlock(run_helper: RunHelper, test_name=None
         raise TestError(
             f"The stake should be decreased by {amount_to_withdraw}. Expected {current_stake - amount_to_withdraw}, got {result[0].get('total_stake')}"
         )
+
+
+@test_case
+def test_stake_request_commisson(run_helper: RunHelper, test_name=None):
```
Comment (xbtmatt) [inline crates/aptos/e2e/cases/stake.py:362]:
```suggestion
def test_stake_request_commission(run_helper: RunHelper, test_name=None):
```

File: crates/aptos/e2e/cases/stake.py [hunk 2]
```diff
@@ -356,3 +356,63 @@ def test_stake_withdraw_stake_after_unlock(run_helper: RunHelper, test_name=None
...
+    # create a new account
+    run_helper.run_command(
+        test_name,
+        [
+            "aptos",
+            "init",
+            "--profile",
+            "request_comisson",
```
Comment (xbtmatt) [inline crates/aptos/e2e/cases/stake.py:370]:
```suggestion
            "request_comission",
```

Comment (xbtmatt) [reply crates/aptos/e2e/cases/stake.py:370]: I won't do the rest of them, there's about 6 more haha

**Review by xbtmatt — APPROVED** [review]

File: crates/aptos/e2e/cases/stake.py (hunk 2 above)
Comment (0xjinn) [reply crates/aptos/e2e/cases/stake.py:370]: oops good catch!

**Review by gregnazario — APPROVED** [review]

**Pull request merged**